.github/workflows/npm-cd.yml

```diff
@@ -151,6 +152,158 @@ jobs:
                   name: ${{ matrix.build.TARGET }}
                   path: ./node/bin
                   if-no-files-found: error
+            
+            - name: Reset repository
```

Comment:
looks like it's being called 5 times over the actions, can you move it to a different action?